### ISSUE TITLE
Heads up about .php_cs, .travis.yml, .eslintignore, and .eslintrc files

### ISSUE THREAD

**Issue by tobscure** [maelsoucaze/flarum#14]
You don't really need those files in this project – they're quite specific to the Flarum project itself. Specifically:

* **.php_cs** is used for fixing PHP coding style, which is not important for a locale extension. It also adds copyright headers to every PHP file, which I see you've added manually – these are not necessary for third-party extensions because this work is in fact not a part of the Flarum project, nor copyrighted to me :)
* **.travis.yml** is not needed
* **.eslintignore** and **.eslintrc** are for checking the style of JavaScript files, of which there are none in this extension

As a general rule, if you don't know what it does, leave it out :)

Regardless, I have previously mentioned to you that the whole structure for locale extensions will be getting revamped soon, and that it'll be best to regenerate the extension from scratch, so don't worry too much about this. Just thought I'd mention it for your own interest :)

**Comment by maelsoucaze** [comment]
Thanks for the tips, I'm removing these. I checked the files to be sure they're not breaking anything before I uploaded them. I had a chance to upload useless files, and that's the case, too bad for me. Mmh, at least I won an explaination, and it's always interesting!

I'm aware I'll have to revamp the whole extension soon, but I'm trying to maintain it for French users who want to test Flarum in the actual beta.

Concerning headers, I'm also aware they're necessary for third-party content, but it helps to compare changes between existing official extensions for now as the code is moving fast. I will replace them with appropriates ones when the coding will be slowest.
